CIP-0072/README.md

```diff
@@ -32,25 +32,21 @@ Also, having this formalisation facilitates any actor in the ecosystem to index
 - **metadata claim** - Generically any attempt to map off-chain metadata to an on-chain subject. This specification looks at dApp specific metadata claims.
 - **client** - Any ecosystem participant which follows on-chain data to consume metadata claims (i.e. dApp stores, wallets, auditors, block explorers, etc.).
 - **dApp Store** - A dApp aggregator application which follows on-chain data looking for and verifying dApp metadata claims, serving their users linked dApp metadata.
-- **publishers** - Entities which publish metadata claims on-chain, in the case of dApps the publishers are likely the dApp developer(s).
-- **auditors** - These are clients which maintain lists of trusted entities and metadata servers, checking metadata claims against these. 
+- **publishers** - Entities which publish metadata claims on-chain, in the case of dApps the publishers are likely the dApp developer(s). 
 
 ### **Developers / Publishers**
 Developers and publishers of dApps can register their dApps by submitting a transaction on-chain that can be indexed and verified by stores, auditors and other ecosystem actors.
 
 ### **Stores / Auditors**
 Stores and auditors should be able to follow the chain and find when a new dApp registration is **anchored** on-chain. They should then perform *integrity* and *trust* validations on the dApp's certificate and metadata.
 
-##### **Targetted Releases**
```

Comment:
Why remove this?

Reply:
@matiwinnetou 
Two points:
1. I was thinking that it might not be possible to do taking into account the current CIP-72 process. t doesn't seem to be a way in the current CIP to mark that this particular release is for a particular consumer (a DApp store, auditor, etc.) - and maybe it is too early to introduce this.
2. Now, it feels that this part deviates from the current CIP-72 process that aims to let DApp developers set up and maintain a single off-chain JSON (as you can see we unify things and create predefined lists to avoid potential mess) that cable to cater to various consumers (e.g. DApp stores, auditors, other DApps, etc.). So, to keep things simple in the MVP version of the CIP, it feels it is better to move this stuff completely or move it to the bottom of the CIP and convey it as a discussion point to monitor feedback and demand for it.

Reply:
Depends how you interpret targeted releases, can you provide your interpretation?